google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentRunConfigurationEditor.java

```diff
@@ -214,7 +217,11 @@ public File get() {
     versionOverrideCheckBox.addItemListener(
         new CustomFieldOverrideListener(versionOverrideCheckBox, versionIdField));
 
-    environmentLabel.setText(environment.localizedLabel());
+    if (environment.isStandard()) {
+      environmentLabel.setText(environment.localizedLabel());
+    } else {
+      environmentLabel.setText(getEnvironmentDisplayableLabel());
+    }
```

Comment:
Agreed we should get this in. I'll review now.

On Thu, Jun 23, 2016 at 10:47 AM, Etan Shaul <notifications@github.com>
wrote:

> In
> google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentRunConfigurationEditor.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/753#discussion_r68248530>
> :
>
> > @@ -214,7 +217,11 @@ public File get() {
> >      versionOverrideCheckBox.addItemListener(
> >          new CustomFieldOverrideListener(versionOverrideCheckBox, versionIdField));
> >
> > -    environmentLabel.setText(environment.localizedLabel());
> > +    if (environment.isStandard()) {
> > +      environmentLabel.setText(environment.localizedLabel());
> > +    } else {
> > +      environmentLabel.setText(getEnvironmentDisplayableLabel());
> > +    }
>
> Oh never mind. I see what you are saying. Good idea.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/753/files/9c3f15148916d3153bee00492314aabb77282850#r68248530>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5HQ3z74CaftRWWSP-XSqR0cFv0D8vks5qOpyRgaJpZM4I8IGj>
> .
>